src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const marriedPrefix = sex === 'm' ? ' is my wife' : ' is my husband';
```

Comment:
` is my x` is repeated here, you can just return the spouse here and combine it with the text in the `p` tag